json/go.mod

```diff
@@ -0,0 +1,3 @@
+module example.com/json
```

Comment:
Please squash all the commits into one

Reply:
Corrected

Reply:
I think that "example.com/json" isn't the best module name for our JSON API :)